packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-items/jurisdiction-modal-card/jurisdiction-card.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Icon, Text, Popover } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { jurisdiction_contents } from 'Constants/jurisdiction-contents';
+import { TJurisdictionCardProps } from 'Containers/props.types';
+import VerificationStatusBanner from './verification-status-banner';
+
+const JurisdictionCard = ({
+    account_type,
+    disabled,
+    jurisdiction_selected_shortcode,
+    financial_available_accounts,
+    setJurisdictionSelectedShortcode,
+    synthetic_available_accounts,
+    type_of_card,
+}: TJurisdictionCardProps) => {
+    const card_classname = `cfd-jurisdiction-card--${account_type}`;
+    const number_of_synthetic_accounts_to_be_shown = synthetic_available_accounts?.length;
+    const number_of_financial_accounts_to_be_shown = financial_available_accounts?.length;
+
+    const [number_of_cards] = React.useState(
+        account_type === 'synthetic'
+            ? number_of_synthetic_accounts_to_be_shown
+            : number_of_financial_accounts_to_be_shown
+    );
+
+    const cardSelection = (cardType: string) => {
+        if (jurisdiction_selected_shortcode === cardType) {
+            setJurisdictionSelectedShortcode('');
+        } else {
+            setJurisdictionSelectedShortcode(cardType);
+        }
+    };
+
+    const Checkmark = () => (
+        <Icon icon='IcCheckmark' className={`${card_classname}__bullet-wrapper--checkmark`} color={'green'} size={18} />
+    );
+
+    const one_or_two_cards = number_of_cards === 1 || number_of_cards === 2;
+
+    return (
+        <>
+            <div
+                className={classNames(card_classname, {
+                    [`${card_classname}--selected`]: jurisdiction_selected_shortcode === type_of_card,
+                })}
+                onClick={disabled ? () => undefined : () => cardSelection(`${type_of_card}`)}
```

Comment:
There's no need for template literals here
```suggestion
                onClick={disabled ? () => undefined : () => cardSelection(type_of_card)}
```